src/ResourceManager/AzureBatch/documentation/upcoming-breaking-changes.md

```diff
@@ -25,20 +25,21 @@
     https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes/breaking-change-template.md
 -->
 
-# Upcoming Breaking Changes   
-
-## Release 5.0.0 - November 2017
-
-- `PSCloudPool` `ResizeError` was replaced by `ResizeErrors`.
-- `PSJobPreparationTaskExecutionInformation`, `PSJobReleaseTaskExecutionInformation`, `PSStartTaskInformation`, `PSTaskExecutionInformation`, and `PSSubtaskInformation` 
-  no longer have a `SchedulingError` property, they instead have a `FailureInformation` property. `FailureInformation` is returned any time there is a task failure.
-  This includes all previous scheduling error cases, as well as nonzero task exit codes, and file upload failures from the new output files feature.
+# Upcoming Breaking Changes
+
+### Changes to objects
+- Renamed `ResizeError` to `ResizeErrors` on `PSCloudPool`, and it is now a collection.
+- Renamed the `Name` property on `PSNodeFile` to `Path`.
+- Renamed `PSTaskSchedulingError` to `PSTaskFailureInformation`.
+- Renamed the `SchedulingError` property on `PSJobPreparationTaskExecutionInformation`, `PSJobReleaseTaskExecutionInformation`, `PSStartTaskInformation`, `PSSubtaskInformation`, and `PSTaskExecutionInformation` to `FailureInformation`.
+  - `FailureInformation` is returned any time there is a task failure. This includes all previous scheduling error cases, as well as nonzero task exit codes, and file upload failures from the new output files feature.
+- Renamed the `SchedulingError` property on `PSExitConditions` to `PreProcessingError`.
 - `PSMultiInstanceSettings` constructor no longer takes a required `numberOfInstances` parameter, instead it takes a required `coordinationCommandLine` parameter.
 
 The following cmdlets were affected this release:
 
 **New-AzureBatchCertificate**
-- Parameter "Password" being replaced in favor of a Secure string
```

Comment:
Please add explnantions of the change form 'runelevated' to 'UserIdentity'

Reply:
Done -- I think I've covered this now both as a parameter to exisiting cmdlets as well as on the returned object from `Get-AzureBatchTask`